Initial draft for #MTTDOC 351 (#313)
Unity-Technologies/com.unity.multiplayer.docs: docs/mlapi-basics/networkobject.md

## Ownership

> Each `NetworkObject` is owned by a specific client. This can be any client of the server.

Reviewer: "or" is intentional here, not a grammar mistake. Any client OR the server can own `NetworkObject`s.

We could try to rephrase like:
```suggestion
Each `NetworkObject` has a specific owner which can be any client or the server itself.
```
